.travis.yml

```diff
@@ -43,7 +47,8 @@ install:
   - case "$BUILD" in
       stack)
         $STACK setup --no-terminal;
-        $STACK build --only-snapshot --no-terminal;;
+        $STACK build --only-snapshot --no-terminal;
+        $STACK build --no-terminal happy;;
```

Comment:
Yeah, I think this is a bash-ism for case statements (or at least how Travis uses it).